src/openzaak/conf/ci.py

```diff
@@ -20,6 +20,7 @@
     # See: https://github.com/jazzband/django-axes/blob/master/docs/configuration.rst#cache-problems
     "axes": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"},
     "oidc": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
+    "kic_sync": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
```

Comment:
I find it odd that suddenly we need this while the mechanism already exists for external URLS with Documents API + Zaken/Besluiten API